AODN/MHL/process_MHLsst_from_txt.py

```diff
@@ -248,6 +248,6 @@ def parse_nc_attribute(input_netcdf_file_path, output_nc_obj):
 
     """
 
-    for txtfile in glob.glob('/vagrant/tmp/MHL/TXTFILES/*.TXT'):
+    for txtfile in glob.glob('/vagrant/tmp/MHL/TXTFILES/*_new.txt'):
```

Comment:
It would be cleaner to parse the input path from the command line, rather than hard-coding it here. It only takes a couple of lines with the `argparse` module. See e.g. [here](https://github.com/aodn/data-services/blob/master/AODN/NSW-OEH/process_zip.py#L325)